platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ExportMigrationManagerImpl.java

```diff
@@ -144,6 +146,15 @@ public void doExport(String productBranding, String productVersion) {
     Validate.notNull(productBranding, "invalid null product branding");
     Validate.notNull(productVersion, "invalid null product version");
     final String ddfHome = System.getProperty("ddf.home");
+    final Properties systemProperties = System.getProperties();
+    String customSystemPropertiesFile = "";
+    try {
+      customSystemPropertiesFile =
+          new String(
+              Files.readAllBytes(MigrationContextImpl.METADATA_CUSTOM_SYSTEM_PROPERTIES_PATH));
+    } catch (IOException e) {
+      LOGGER.error("Could not read custom system properties file.");
```

Comment:
❗️ This should not just be logged blindly. If you cannot read it for whatever reason then an error should be generated.